src/main/java/teammates/storage/api/FeedbackSessionsDb.java

```diff
@@ -34,47 +32,24 @@
 
     public static final String ERROR_UPDATE_NON_EXISTENT = "Trying to update non-existent Feedback Session : ";
 
+    /**
+     * Gets a list of feedback sessions that have start/end time laid between {@code rangeStart} and {@code rangeEnd}.
+     */
     public List<FeedbackSessionAttributes> getAllOpenFeedbackSessions(Instant rangeStart, Instant rangeEnd) {
-        List<FeedbackSessionAttributes> list = new LinkedList<>();
-
-        // To retrieve legacy data where local dates are stored instead of UTC
-        // TODO: remove after all legacy data has been converted
-        Instant start = rangeStart.minus(Duration.ofHours(25));
-        Instant end = rangeEnd.plus(Duration.ofHours(25));
```

Comment:
This was to account for possible timezone offsets before. Although I trust the timezone-handling code, can confirm that it's safe to remove these?

Reply:
After running script in #9084, all entities should be stored in UTC right now.